### ISSUE TITLE
Can I host COPS on Github or Netlify with library in cloud?

### ISSUE THREAD

**Issue by Aspiring-Writer** [seblucas/cops#513]
Hello!

I just want to know if it would be possible for me to run the COPS server on GitHub Pages or Netlify with my Calibre library in a cloud service (in my case, Mega).

I don't know PHP, so I don't want to mess around with this unless it will work. In theory, I don't see why the server itself wouldn't work, but I don't know about the cloud library. Why can't I find anything on this? Lol

Thanks in advance!

**Comment by marioscube** [comment]
You need to be able to "mount" the cloud service on a directory in the webserver.
So that depends on:
 - can you mount samba or NFS or ? on the webserver.
 - does the cloud server provide samba or NFS or ? access?

I don't know.

**Comment by marioscube** [comment]
You need to be able to point COPS to a calibre library.

$config['calibre_directory'] = './Calibrelibrary/';

if you are able to do that it might work.

**Comment by Aspiring-Writer** [comment]
Yeah, it doesn't work :( I think its my cloud service though. Thanks anyway

**Comment by mikespub** [comment]
Github Pages or Netlify only support static files, so you would need to use something like https://github.com/calibre2opds/calibre2opds or a more modern equivalent to generate the files first

For COPS you need to have some backend server that can handle PHP scripts. Starting with release 2.7.1 the Calibre library itself can be in the cloud - see #506
